### PR TITLE
chore(dtfs2 none): fix mock data loader tidy up on failure

### PR DESCRIPTION
**Pull request by AlexBramhill**
## Introduction :pencil2:
Updated mock data loader to wipe on error, so there is no need to manually delete database

### CODE REVIEW

File: utils/mock-data-loader/re-insert-mocks.js [hunk 1]
```diff
@@ -17,22 +17,34 @@ const { logger, LOGGER_COLOURS } = require('./helpers/logger.helper');
...
+    await insertMocks(portalToken);
+    await insertMocksGef(portalToken);
+
+    const tfmToken = await createAndLogInAsInitialTfmUser();
+    await insertMocksTfm(tfmToken);
+
+    await deleteInitialTFMUser(tfmToken);
+    await deleteInitialUser(portalToken);
+  } catch (error) {
+    logger.warn('An error occurred, attempting to clean all tables');
```
Comment (abhi-markan) [inline utils/mock-data-loader/re-insert-mocks.js:36]:
```suggestion
    logger.warn('An error occurred, attempting to clean all tables %o', error);
```

File: utils/mock-data-loader/re-insert-mocks.js [hunk 2]
```diff
@@ -17,22 +17,34 @@ const { logger, LOGGER_COLOURS } = require('./helpers/logger.helper');
...
+    try {
+      const portalToken = await createAndLogInAsInitialUser();
+      await cleanAllTablesPortal(portalToken);
+      await cleanAllTablesGef(portalToken);
+      await cleanAllTablesTfm();
+    } catch {
+      logger.warn('Not all tables could be cleared. Consider manually clearing your database before retrying');
+    }
+    logger.error('The following error occurred while attempting to reinsert mocks:');
```
Comment (abhi-markan) [inline utils/mock-data-loader/re-insert-mocks.js:45]: LOC :36 and :45 both being printed out on the console can be confusing, perhaps we should look into merging into one statement please. Thanks

Comment (AlexBramhill) [reply utils/mock-data-loader/re-insert-mocks.js:45]: image.png

Comment (AlexBramhill) [reply utils/mock-data-loader/re-insert-mocks.js:45]: image
image

File: utils/mock-data-loader/re-insert-mocks.js (hunk 1 above)
Comment (AlexBramhill) [reply utils/mock-data-loader/re-insert-mocks.js]: I'll answer why I'm not doing this below :)

File: utils/mock-data-loader/re-insert-mocks.js (hunk 2 above)
Comment (AlexBramhill) [reply utils/mock-data-loader/re-insert-mocks.js:45]: I hope it's not too confusing -- I wanted a clear way of denoting where the clear up begins vs the main script ends after throwing an error... I've updated it to use `error`, so it all should read clearly. Let me know what you think.

For reference these are the logs for a failure:
image

And these are the logs for a failure, and a subsequent failure on the clean up
image

**Review by abhi-markan — APPROVED** [review]

**Pull request merged**